src/core/stakes.zig

```diff
@@ -0,0 +1,1787 @@
+const builtin = @import("builtin");
```

Comment:
I think you meant to delete stake.zig when you added this file, but it's still there.